lib/indico.js

```diff
@@ -134,7 +134,9 @@ var make_request = function (name, data, batch, config) {
 var api_request = function (name, api, batch) {
   return function (data, config) {
     if (api.type === "image") {
-      return image.preprocess(data, api.size, batch).then(function (packaged) {
+      return image.preprocess(data,
+          (api == "fer" && config["detect"]) ? false : api.size, batch
```

Comment:
Going to move this out of the function call and assign to a variable, but aside from that looks good.